tests/foreman/cli/test_discoveredhost.py

```diff
@@ -130,72 +130,6 @@ def test_positive_pxe_based_discovery(self):
             host = self._assertdiscoveredhost(hostname)
             self.assertIsNotNone(host)
 
-    @pytest.mark.tier3
-    def test_positive_pxe_less_with_dhcp_unattended(self):
```

Comment:
this is already contained as a step in
`test_positive_provision_pxeless_bios_syslinux`